CalibCalorimetry/EcalLaserCorrection/plugins/EcalLaserCorrectionService.cc

```diff
@@ -62,6 +65,8 @@ EcalLaserCorrectionService::EcalLaserCorrectionService(const edm::ParameterSet&
       .setConsumes(apdpnToken_)
       .setConsumes(linearToken_);
 
+  deltat_safety_ = fConfig.getUntrackedParameter<int>("deltat_safety", 0);
```

Comment:
should be changed for tracked (?)
convert seconds to 64bit CMSSW format here:

```suggestion
  deltat_safety_ = fConfig.getParameter<int>("deltat_safety", 0);
  deltat_safety = deltat_safety<<32;
```